### PR TITLE
Test case for generate and confirm paper

### PR DESCRIPTION
**Pull request by vaibhav0320**
It need some fixation that I commented in the code because I was not able to understand how paper is generated.

### CODE REVIEW

File: test/test_course.py [hunk 1]
```diff
@@ -1,30 +1,26 @@
-from flask import json
-
-from flaskapp import models
 from test.main.base_classes import BaseUser
 from test.main.utils import test_post_request
```
Comment (Niraj-Kamdar) [inline test/test_course.py:2]: This isn't regarding generate_paper so, remove this changes

**Comment by Niraj-Kamdar** [comment]
There isn't any questions that satisfy given constraints

**Comment by Niraj-Kamdar** [comment]
Ask @jeel2308 to give you valid question and mark template that can generate paper

**Comment by vaibhav0320** [comment]
Added comprehensive , knowledge and application mcq and sub question for easy, medium and hard still can't generate paper

**Comment by Niraj-Kamdar** [comment]
That means Constraints aren't satisfying

**Comment by Niraj-Kamdar** [comment]
Fixed

**Pull request closed without merging**